rocAL/rocAL/source/meta_data/bounding_box_graph.cpp

```diff
@@ -138,20 +131,20 @@ inline int find_best_box_for_anchor(unsigned anchor_idx, const std::vector<float
     return best_idx;
 }
 
-void BoundingBoxGraph::update_box_encoder_meta_data(std::vector<float> *anchors, pMetaDataBatch full_batch_meta_data, float criteria, bool offset, float scale, std::vector<float>& means, std::vector<float>& stds)
+void BoundingBoxGraph::update_box_encoder_meta_data(std::vector<float> *anchors, pMetaDataBatch full_batch_meta_data, float criteria, bool offset, float scale, std::vector<float>& means, std::vector<float>& stds, float *encoded_boxes_data, int *encoded_labels_data)
 {
     #pragma omp parallel for
     for (int i = 0; i < full_batch_meta_data->size(); i++)
     {
-        BoundingBoxCord *bbox_anchors = reinterpret_cast<BoundingBoxCord *>(anchors->data());
+        BoundingBoxCordInfo *bbox_anchors = reinterpret_cast<BoundingBoxCordInfo *>(anchors->data());
         auto bb_count = full_batch_meta_data->get_labels_batch()[i].size();
         std::vector<BoundingBoxCord> bb_coords;
         Labels bb_labels;
         bb_labels.resize(bb_count);
         bb_coords.resize(bb_count);
         memcpy(bb_labels.data(), full_batch_meta_data->get_labels_batch()[i].data(), sizeof(int) * bb_count);
```

Comment:
I think we need not use the memcopy in these 2 lines here, since we are not updating these values we can directly make use of the bbox and labels without storing into another buffer